app/javascript/src/global/_common.scss

```diff
@@ -170,10 +170,20 @@ strong {
             @media only screen and (max-width: 767px) {
                 display: none !important;
             }
-        }   
+        }
     }
 }
 
+.auction-newsletter-link {
+    margin-left: 0.75em !important;
```

Comment:
I understand it might be required by the current framework, but I still wonder if there is any chance to avoid `!important` (way too many of them :( ). It will be a nightmare to maintain such CSS later.

Reply:
I agree. Eliminated one !important tag. In overall, I don't see any other reasonable methods on how to overwrite these values because fair amount of rules are forced like this as well. But as long as overrides are in single file(s) systematically, I'm not concerned.